sdk/nodejs/arn.ts

```diff
@@ -12,8 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import {ManagedPolicy} from "./iam";
 /**
  * An ARN is an Amazon Resource Name, and uniquely identifies a region globally across all accounts and regions.
  */
-export type ARN = string;
+export type ARN = string | ManagedPolicy;
```

Comment:
I didn't look into the details here. What was the reason for this change?

Reply:
ARN is a "relaxed enum". The existing [ManagedPolicies](https://github.com/pulumi/pulumi-aws/blob/master/sdk/nodejs/iam/managedPolicies.ts) have ARN as their type. Schematizing those as ManagedPolicy means that the type is `ManagedPolicy` instead of `ARN` so `ARN` can now be `ManagedPolicy` or any odd string.